app/renderer/components/common/browserButton.js

```diff
@@ -16,12 +16,14 @@ class BrowserButton extends ImmutableComponent {
       this.props.subtleItem && [styles.browserButton_default, styles.browserButton_subtleItem],
       this.props.extensionItem && styles.browserButton_extensionItem,
       this.props.groupedItem && styles.browserButton_groupedItem,
-      this.props.notificationItem && styles.browserButton_notificationItem
+      this.props.notificationItem && styles.browserButton_notificationItem,
+      // actionItem is just subtleItem with a blue background
+      this.props.actionItem &&
+        [styles.browserButton_default, styles.browserButton_subtleItem, styles.browserButton_actionItem]
```

Comment:
I would move the 3 lines above to a line between L16-L17, `subtleItem` and `extensionItem`.